### PR TITLE
Check system image type

### PR DESCRIPTION
**Pull request by sn-o-w**
The idea came after seeing this: https://github.com/LSPosed/MagiskOnWSALocal/issues/615#issuecomment-1626997122

### CODE REVIEW

File: scripts/build.sh [hunk 1]
```diff
@@ -639,16 +639,15 @@ if [ "$GAPPS_BRAND" != 'none' ]; then
...
-
-if [[ "$WSA_MAJOR_VER" -ge 2304 ]]; then
+echo "Convert vhdx to RAW image"
+vhdx_to_raw_img "$WORK_DIR/wsa/$ARCH/system_ext.vhdx" "$WORK_DIR/wsa/$ARCH/system_ext.img" || abort
+vhdx_to_raw_img "$WORK_DIR/wsa/$ARCH/product.vhdx" "$WORK_DIR/wsa/$ARCH/product.img" || abort
+vhdx_to_raw_img "$WORK_DIR/wsa/$ARCH/system.vhdx" "$WORK_DIR/wsa/$ARCH/system.img" || abort
+vhdx_to_raw_img "$WORK_DIR/wsa/$ARCH/vendor.vhdx" "$WORK_DIR/wsa/$ARCH/vendor.img" || abort
+echo -e "Convert vhdx to RAW image done\n"
+
+SYSTEM_IMAGE_TYPE=$(blkid -o value -s TYPE "$WORK_DIR/wsa/$ARCH/system.img")
```
Comment (YT-Advanced) [inline scripts/build.sh:649]: I think you should move this command to a seperate func. You can check my code:
https://github.com/YT-Advanced/WSA-Script/blob/096213de2a2ba19e1a6088d1ab132cbd3fd9d567/scripts/build.sh#L134

Comment (YT-Advanced) [reply scripts/build.sh:649]: Easier manager, etc

**Comment by Erisa** [comment]
This PR appears to resolve #670

**Comment by ningyuv** [comment]
> > Easier manager, etc
> 
> What in the current code is not "easy to manage"? 😂

> This PR appears to resolve #670

So this PR appears to "easy to manage"

**Pull request closed without merging**